docs/src/main/paradox/scala/http/06extensions.md

```diff
@@ -0,0 +1,3 @@
+# 6. Extensions
+
+This page list well-known third-part extensions of Akka HTTP. See the Akka HTTP tag on scaladex for more info. // TODO: add link 
```

Comment:
Link to https://akka.io/community/  

or https://index.scala-lang.org/search?topics=akka-http